jwst/tweakreg/tweakreg_step.py

```diff
@@ -27,9 +27,11 @@ class TweakRegStep(Step):
     spec = """
         # Source finding parameters:
         save_catalogs = boolean(default=False) # Write out catalogs?
-        catalog_format = string(default='ecsv')   # Catalog output file format
-        kernel_fwhm = float(default=2.5)    # Gaussian kernel FWHM in pixels
-        snr_threshold = float(default=10.0)  # SNR threshold above the bkg
+        catalog_format = string(default='ecsv') # Catalog output file format
+        kernel_fwhm = float(default=2.5) # Gaussian kernel FWHM in pixels
+        snr_threshold = float(default=10.0) # SNR threshold above the bkg
+        brightest = int(default=None) # Keep top ``brightest`` objects
+        peakmax = float(default=None) # Filter out objects with pixel values >= ``peakmax``
```

Comment:
The /pipeline/tweakreg.cfg file should also be updated to include these parameter changes (and to make sure they're consistent with one another).